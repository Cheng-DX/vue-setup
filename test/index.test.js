import { expect, test } from 'vitest'

function randomXData(seed) {
  let data = [];
  for (let i = 0; i < seed; i++) {
    data.push('X' + (i + 1))
  }
  return data;
}

function randomSeries(seed) {
  let series = []
  for (let i = 0; i < seed; i++) {
    let data = []
    for (let i = 0; i < seed; i++) {
      data.push(Math.floor(Math.random() * 1000))
    }
    series.push({
      name: 'Random Series' + (i + 1),
      data,
      type: Math.random() > 0.5 ? 'line' : 'bar',
      smooth: true,
      emphasis: {
        focus: 'series'
      },
    })
  }
  return series;
}


import indexVue from '../src/'
test.concurrent('random', () => {
  
});