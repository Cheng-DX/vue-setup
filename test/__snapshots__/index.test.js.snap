// Vitest Snapshot v1

exports[`random > text 1`] = `
"
  function randomXData(seed) {
    let data = [];
    for (let i = 0; i < seed; i++) {
      data.push('X' + (i + 1))
    }
    return data;
  }"
`;
